crest/Assets/Crest/Crest-Examples/BoatDev/Scripts/BoatAlignNormal.cs

```diff
@@ -32,6 +32,8 @@ public class BoatAlignNormal : FloatingObjectBase
     public float _boyancyTorque = 8f;
     [Tooltip("Approximate hydrodynamics of 'surfing' down waves."), Crest.Range(0, 1)]
     public float _accelerateDownhill = 0f;
+    [Tooltip("Clamps the buoyancy force to this value. Useful for handling fully submerged objects. Use Infinity to disable.")]
```

Comment:
Does the user need to enter 'Inf' or does Infinity work? If the former probably worth giving this as instruction as I would 
not know how to put infinity into a float field.

Reply:
Either _Inf_ or _Infinity_. _Inf_ will expand to _Infinity_ when field loses focus. The actually value is _Infinity_.

I've change that sentence to "Enter 'Infinity' to disable". Inf is just a shortcut which users can work out for themselves through Unity.